SCons/Environment.xml

```diff
@@ -1103,6 +1103,9 @@ that runs a build,
 updates a file,
 and runs the build again,
 all within a single second.
+<literal>content-timestamp</literal>
```

Comment:
Need to add a blurb above for MD5 that it should be called content.. like you have here for MD5-Timestamp.

Reply:
Given that we are deprecating MD5 and MD5-timestamp, I renamed the existing options and removed mentions of there being synonyms.

Reply:
Deprecating doesn't mean removing from docs at this point. They should still remain as they're in use with notes that they are being deprecated.